src/components/coin-icon/CoinIconFallback.js

```diff
@@ -55,6 +57,8 @@ function WrappedFallbackImage({
       style={{ overflow: 'hidden' }}
     >
       <FallbackImage
+        as={eth && Image}
```

Comment:
for this I used to use a pattern  like this:

```tsx
function passIf<T extends unknown>(condition: boolean | undefined, value: T): T | undefined {
  if (condition) {
    return value;
  }

  return undefined;
}

<FallbackImage
  as={passIf(eth, Image)}
/>
```

It still passes the value as is or undefined, but do not mess up with the props which expect the value to exist or be undefined.
Also, sometimes it could break the logic inside the component expects it to be undefined.

Reply:
this not ts file so I expect this is the same as 
```suggestion
        as={eth ? Image : undefined}
```
?

Reply:
Yes, this is the same. We can just use ternary here but I found myself using over and over that passIf in projects since it's just handy. 
It's ok to use ternary here but I'll leave you with that passIf idea :)